MediaAdminBundle/Resources/public/coffee/SuperboxView.coffee

```diff
@@ -184,10 +174,6 @@ SuperboxView = OrchestraView.extend(
     $('#image-loader').hide()
     @showPreview(format)
 
-  addSelect2OnForm: ->
```

Comment:
because of OrchestraView->onOrchestraViewReady->activateForm->activateSelect2 @itkg-ngilain